app/utils/tools.py

```diff
@@ -190,28 +181,36 @@ def get_file_from_data(
     directory: str,
     filename: str,
     default_asset: str,
-) -> FileResponse:
+):
     """
     If there is a file with the provided filename in the data folder, return it. The file extension will be inferred from the provided content file.
     > "data/{directory}/{filename}.ext"
     Otherwise, return the default asset.
 
-    The filename should be a uuid.
-
     WARNING: **NEVER** trust user input when calling this function. Always check that parameters are valid.
     """
-    if not uuid_regex.match(filename):
-        hyperion_error_logger.error(
-            f"get_file_from_data: security issue, the filename is not a valid UUID: {filename}. This mean that the user input was not properly checked.",
-        )
-        raise ValueError("The filename is not a valid UUID")
-
     for filePath in Path().glob(f"data/{directory}/{filename}.*"):
         return FileResponse(filePath)
 
     return FileResponse(default_asset)
 
 
+def delete_file_from_data(
+    directory: str,
+    filename: str,
+    extension: str,
+):
+    """
+    If there is a file with the provided filename in the data folder, delete it. The file extension must be precised.
+    > "data/{directory}/{filename}.{extension}"
+    """
```

Comment:
Could you add the UUID regex verification and the comment explaining to never thrust the user input?

Reply:
I don't know why but it doesn't work with `"data/{directory}/{filename}.*"` but it works like this so ...